assets/wire/components/fields/ExpiryDateLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+import {get} from 'lodash';
+
+export function ExpiryDateLabel ({item, filterGroupLabels}: any) {
+    if (item.expiry == null) {
+        return null;
+    }
+
+    const label = get(filterGroupLabels, 'expiry', gettext('Expiry Date'));
+    const value = item.expiry;
+    const text = label + ': ' + value;
```

Comment:
the value should be formatted using `utils.formatDate`

Reply:
can u give me more informations? :D

Reply:
we don't want to display in the UI the server format `2023-12-07T09:53:00` , we have a function for that in utils:
```ts
import {formatDate} from 'utils';
value = formatDate(item.expiry);
```